coderedcms/fields.py

```diff
@@ -11,3 +12,16 @@ def __init__(self, *args, **kwargs):
     def formfield(self, **kwargs):
         kwargs['widget'] = ColorPickerWidget
         return super(ColorField, self).formfield(**kwargs)
+
+
+class ScriptField(models.TextField):
+    def __init__(self, *args, **kwargs):
+        super(ScriptField, self).__init__(*args, **kwargs)
+
+    def formfield(self, **kwargs):
+        kwargs['widget'] = Textarea(attrs={
+            'rows': 15,
+            'cols': 10,
```

Comment:
I don't think cols is necessary... I believe it fills to 100% width within wagtail admin.

Reply:
done